src/aeonlib/ocs/target_models.py

```diff
@@ -62,33 +52,48 @@ class NonSiderealTarget(BaseModel):
         "MPC_COMET",
     ]
     """The Target scheme to use"""
-    epochofel: Annotated[int, Ge(10_000), Le(100_000)]
+    epochofel: Annotated[float, Ge(10_000), Le(100_000)]
     """The epoch of the orbital elements (MJD)"""
-    orbinc: Annotated[int, NonNegativeFloat, Le(180.0)]
+    orbinc: Annotated[float, NonNegativeFloat, Le(180.0)]
     """Orbital inclination (angle in degrees)"""
-    longascnode: Annotated[int, NonNegativeFloat, Le(360.0)]
+    longascnode: Annotated[float, NonNegativeFloat, Le(360.0)]
     """Longitude of ascending node (angle in degrees)"""
-    longofperih: Annotated[int, NonNegativeFloat, Le(360.0)] | None = None
-    """Longitude of perihelion (angle in degrees)"""
-    argofperih: Annotated[int, NonNegativeFloat, Le(360.0)]
+    argofperih: Annotated[float, NonNegativeFloat, Le(360.0)]
     """Argument of perihelion (angle in degrees)"""
-    meandist: float
-    """Mean distance (AU)"""
-    perihdist: float | None = None
-    """Perihelion distance (AU)"""
     eccentricity: NonNegativeFloat
     """Eccentricity of the orbit"""
-    meanlong: float | None = None
-    """Mean longitude (angle in degrees)"""
+    meandist: Annotated[float, NonNegativeFloat]
+    """Semi-major axis (AU)""" # Not Comet
     meananom: Annotated[float, NonNegativeFloat, Le(360.0)]
     """Mean anomaly (angle in degrees)"""
+    perihdist: Annotated[float, NonNegativeFloat] | None = None
+    """Perihelion distance (AU)""" # Comet Only
+    epochofperih: Annotated[float, Le(240_000), Le(100_000)] | None = None
+    """Epoch of perihelion (MJD)""" # Comet Only
     dailymot: float | None = None
-    """Daily motion (angle in degrees)"""
-    epochofperih: Annotated[float, Ge(361), Le(240_000), Le(100_000)] | None = None
-    """Epoch of perihelion (MJD)"""
+    """Daily motion (angle in degrees)""" # Major Planet Only
+    altitude: Annotated[float, NonNegativeFloat, Le(90)] | None = None
+    """Altitude of this Target in decimal degrees""" # Satellite Only
+    azimuth: Annotated[float, NonNegativeFloat, Le(360)] | None = None
+    """Azimuth of this Target in decimal degrees east of North""" # Satellite Only
+    diff_altitude_rate: float | None = None
+    """Differential altitude rate (arcsec/s)""" # Satellite Only
+    diff_azimuth_rate: float | None = None
+    """Differential azimuth rate (arcsec/s)""" # Satellite Only
+    diff_epoch: float | None = None
+    """Reference time for non-sidereal motion (MJD)""" # Satellite Only
+    diff_altitude_acceleration: float | None = None
+    """Differential altitude acceleration (arcsec/s^2)""" # Satellite Only
+    diff_azimuth_acceleration: float | None = None
+    """Differential azimuth acceleration (arcsec/s^2)""" # Satellite Only
+    meanlong: float | None = None
+    """Mean longitude (angle in degrees)""" # No idea what this is.
```

Comment:
I don't know what these last two are. They might be related to satelites, but I don't know how.